packages/create-svelte/bin.js

```diff
@@ -180,7 +176,15 @@ async function main() {
 	console.log(`  ${i++}: ${bold(cyan('npm run dev -- --open'))}`);
 
 	console.log(`\nTo close the dev server, hit ${bold(cyan('Ctrl-C'))}`);
-	console.log(`\nStuck? Visit us at ${cyan('https://svelte.dev/chat')}\n`);
+	console.log(
```

Comment:
this seems a bit verbose to include in the CLI. I'd probably just keep this portion how it was before

Reply:
I like that it's three items (a nice number of sentences), each is short, starts with a one-word-question, and serves a different purpose.

Reply:
I think the one I'm most hesitant about is encouraging people to file issues l. I sort of liked directing people to Discord first because then the community can help us triage a bit whether it's a usage issue or a bug that should be filed

Reply:
Sounds good, I'll remove that item.

Reply:
I'd like the tutorial to be a bit more buttoned up before we start directing people there without caveats. For example if you follow the link here you go to a page that begins

> So far, we've been working on individual components, or groups of components, in isolation

which is a major WTF if you land there without context. Let's revert this to how it was before

Reply:
will push that change, one sec

Reply:
FYI I also mention the tutorial on the intro docs - do you want to remove it from there, too?

Reply:
i think we can link to it, but with caveats (and probably to the first page, not straight to part 2). i'm going to tweak some other word in there anyway so will do that now